src/layout/header/Menu.tsx

```diff
@@ -102,11 +102,11 @@ const Menu: FC<MenuProps> = ({ menuButtonLabel, handleLogout }) => {
                 {t('common:institutions')}
               </MenuItem>
             )}
-            {user.isInstitutionAdmin && (
+            {user.isInstitutionAdmin && user.customerId && (
               <>
                 <MenuItem
                   data-testid="menu-admin-institution-button"
-                  onClick={() => handleClickMenuItem(`/admin-institutions?id=${encodeURIComponent(user.customerId)}`)}>
+                  onClick={() => handleClickMenuItem(`/admin-institutions?id=${encodeURIComponent(user.customerId!)}`)}>
```

Comment:
hva var `!` til slutt igjen?

Reply:
hvis vi vet at vi har den verdien, men TS ikke klarer å oppfatte at vi har sjekket på det

Reply:
aah, det har jeg trengt flere ganger faktisk!
men skal den ikke skjønne det her da, siden det ligger inni en sjekk for at den er true?

Reply:
skulle egentlig skjønt det ja, men gjorde ikke det. Og TS skjønner det hvertfall ikke nå når sjekken gjøres i userReducer 😝

Reply:
nei, da er det ikke tvil lenger 😅